README.md

```diff
@@ -98,6 +98,58 @@ Note: The keys of the `data` object should match with the fields that are set us
 
 Article: https://devapt.com/formspree-alternative-formeasy
 
+## Captcha validation
+
+FormEasy supports multiple captcha providers to allow you to prevent unverified submissions by robots. Each provider is unique and requires a unique configuration. Please refer to the documentation below to enable a specific captcha provider.
+
+### Google reCAPTCHA V2
+
+1. Register a site and get your secret key, and site key: [https://www.google.com/recaptcha/admin/create](https://www.google.com/recaptcha/admin/create)
+
+2. In your apps script file, inside function `doPost, add the following configuration:
+
+```js
+function doPost(req) {
+  // ...
+  FormEasy.setRecaptcha('YOUR_SECRET_KEY'); // To validate reCAPTCHA
+  // ...
+  return FormEasy.action(req); // Mandatory to return action method
+}
+```
+
+> Make sure to set your secret key. You should have it ready from step 1
+
+3. On your website, add the reCAPTCHA library at the end of the `<head>` tag:
+
+```html
+<head>
+  <!-- ... -->
+
+  <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+</head>
+```
+
+4. Add reCAPTCHA input into your form:
+
+```html
+<div class="g-recaptcha" data-sitekey="YOUR_SITE_KEY"></div>
+```
+
+> Make sure to set your secret key. You should have it ready from step 1
```

Comment:
At this we don't need to mention about secret key as it is already mentioned before.

Reply:
Applied. Also removed `> Make sure to set your secret key. You should have it ready from step 1` below step 4

Reply:
I think you have removed this `> Make sure to set your secret key. You should have it ready from step 1` under step 2 as well. 
Keeping it below step 2 will give details about it. You can add it or after merging I'll include it.


So everything seems to be fine, when do you want me to merge and close this PR as you said you want to add more.